scripts/airdropBsxToHdx/getVestingSchedules.ts

```diff
@@ -0,0 +1,121 @@
+// ✅  -> calculate BSX reward for accounts from .json of getData
+
+// ✅ -> calulate 5 000 000 000 proportional to holdings at the time for each holder
+//  ->  output a .json file of all schedules
+
+// ✅  -> start vesting at 9 334 719, Crowd Loan BSX vest
+// ✅  -> end vesting ~22 months after start
+
+const bignumber = require("bignumber.js")
+const airdropData = require("./data/airdropData.json")
+
+type DynamicVestingInfo = {
+    destination: string,
+    schedule: {
+        amountToBeVested: string,
+        start: string,
+        period: string,
+        per_period: string,
+        period_count: string 
+    }
+}
+
+type VestingBatch = DynamicVestingInfo[]
+
+const tenTo12Power = new bignumber('10').pow('12');
+
+const airdropBsxAllocation = 
+    new bignumber('100000000000') // 100b -> total supply of BSX
+    .multipliedBy(new bignumber('0.05'))
+    .multipliedBy(tenTo12Power).toFixed(0)
+// ^^^ 5% of total supply
+
+console.log(
+    '\n total allocation to non-investor, non-founder accounts: ', 
+    new bignumber(airdropBsxAllocation).div(tenTo12Power).toString() + '\n'
+)
+
+const sumHdxBalances = new bignumber(airdropData.totalAllOgHdxBalances)
+
+const bsxAirdroppedPerOneHdx = 
+    new bignumber(airdropBsxAllocation).div(sumHdxBalances)
+
+console.log(
+    '\n amount of BSX tokens you get per one HDX token: ', 
+    bsxAirdroppedPerOneHdx.toString() + '\n'
+)
+
+const leaseStartBlock = '9334719'
```

Comment:
This block number is on Kusama. On Basilisk, this block doesn't exist yet.

Reply:
There is no corresponding block on Basilisk. Kusama block 9334719 was created on 2021-09-22 09:28:00. The first Basilisk block was created 2021-09-22 11:27:54.

Reply:
vesting is based on relay chain blocks https://github.com/galacticcouncil/Basilisk-node/blob/77084c27703432f6c02315ab6bdead3e7dcaf90b/runtime/basilisk/src/lib.rs#L774